apps/server/net/nanows_client.go

```diff
@@ -50,22 +52,25 @@ type WSCallbackMsg struct {
 	Amount  string          `json:"amount"`
 }
 
-func StartNanoWSClient(wsUrl string, callbackChan *chan *WSCallbackMsg) {
+func StartNanoWSClient(wsUrl string, callbackChan *chan *WSCallbackMsg, w *wallet.NanoWallet, newAccountChan <-chan string) {
 	ctx, cancel := context.WithCancel(context.Background())
 	sentSubscribe := false
 	ws := recws.RecConn{}
+
+	addresses, err := w.GetAllAccountAddresses()
+	if err != nil {
+		// Handle error
```

Comment:
I know gpt wrote this, but this should be handled. Maybe modify the function to return an error and consider panicking if there's an error.